src/main/java/Main.java

```diff
@@ -108,4 +119,137 @@ public static void main(String[] args) {
 
 		System.out.println("Player to win: " + winnerName);
 	}
+
+	public static void whatWillICallThis() {
+		int size = DataInfo.getTopPlayers();
```

Comment:
Added this so that the sizes aren't hard coded later on, allows for easier changes